.env

```diff
@@ -13,8 +13,6 @@ REACT_APP_VX_SKIP_BALLOT_PACKAGE_AUTHENTICATION=FALSE
 REACT_APP_VX_SKIP_CAST_VOTE_RECORDS_AUTHENTICATION=FALSE
 REACT_APP_VX_CAST_VOTE_RECORD_OPTIMIZATION_EXCLUDE_ORIGINAL_SNAPSHOTS=TRUE
```

Comment:
@arsalansufi Do we still want these two CVR optimizations as true?

Reply:
Yup, the two CVR optimizations should be kept enabled!